transforms-caching-experiment/remote-cache-experiment-transforms.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Note:
+# This is a script to run the transforms remote caching experiments. You need to update it to use your server url, gradle version (8.9+) and set up the access key with remote cache write permission (you can also export it in the terminal if you wish to share the script with teammates). When executing, you can also supply a different task to run this with, it defaults to help. It will run 3 builds:
+# 1. Init/seed build, that writes to remote build cache
+# 2. Transform caching enabled - pulls from remote cache
+# 3. Transform caching disabled - executes the transforms
+# You can then compare the build times to see how transforms caching affects your builds
+
+# --------- Required -----------
+
+develocityUrl="https://<your-server-url>"
+gradleVersion="8.9" # at least 8.9 is required
```

Comment:
Not to over-engineer this, but could you parse the version from `gradle.properties`? Parsing `./gradlew --version` may actually be easier. It just seems like this would be easy to miss and go unnoticed, throwing off the experiment results. In general, the less the user has to modify the script for it to work, the better. A major reason to automate this is so that it is easily reproducible and correct.